bin/tbg

```diff
@@ -275,7 +275,7 @@ help()
 initCall="$0 $*"
 projectPath="."
 
-pathToegetopt=`which egetopt`
+pathToegetopt="$(which egetopt 2>/dev/null)"
```

Comment:
why the `"` quotes?

Reply:
because it is allowed. But I will remove it.